modules/api-svc/src/FSPIOPEventHandler/handlers/party-info-requested.js

```diff
@@ -56,7 +56,11 @@ module.exports.handlePartyInfoRequestedDmEvt = async (
             bulkId: event.getKey(),
             content: {
                 transferId: event.getTransferId(),
-                partyResult: response,
```

Comment:
I don't think there is a `body` in the response:
https://github.com/mojaloop/sdk-scheme-adapter/blob/16ccfe52f101d38daa397cf3312f33462bf8176c/src/OutboundServer/handlers.js#L467-L475